action_server/docs/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 ## 0.1.4 - 2024-03-20
 
 - Fixed issue in action-server binary build. 
+- Add support parsing Array type in Action Server UI action run view
```

Comment:
This changelog entry is actually in the wrong place (it should be in the `unreleased` section).